pyaedt/edb_core/components.py

```diff
@@ -106,24 +111,24 @@ def _active_layout(self):
 
     @property
     def _cell(self):
-        return self.parent.cell
+       return self.parent.cell
```

Comment:
```suggestion
       return self.parent.cell
```
Should always be 4 spaces indent from method definition line.